examples/complete-http/main.tf

```diff
@@ -49,7 +49,9 @@ module "api_gateway" {
   }
 
   # Domain Name
-  domain_name           = var.domain_name
+  domain_name = var.domain_name
+  # Not strictly required, but handy if you want to create the domain within a specific zone
```

Comment:
```suggestion
```

Reply:
No need to put this in an example.

Reply:
Alright, I only added it after reading the PR template 🙂